src/pages/ModelOfInterest/GraphDisplay/index.tsx

```diff
@@ -64,53 +64,39 @@ const GraphDisplay = () => {
         // Remove existing SVG content
         d3.select("#graph-container").selectAll("*").remove();
 
-        // Specify the bounds for the simulation
-    //const bounds = { x1: 0, y1: 0, x2: width, y2: height };
-
     // Create a simulation with several forces.
     const simulation = d3.forceSimulation(nodes)
-      .force("link", d3.forceLink(links).id((d) => (d as Node).name).distance(50))
-      .force("charge", d3.forceManyBody().strength(-20))
-      .force("center", d3.forceCenter(width / 2, height / 2))
+      .force("link", d3.forceLink(links).id((d) => (d as Node).name))
+      .force("charge", d3.forceManyBody().strength(-250))
+      .force("x", d3.forceX())
+      .force("y", d3.forceY())
+      //.force("link", d3.forceLink(links))
```

Comment:
@pramit-d Please remove the commented code blocks

Reply:
@harimohanr  removed commented code.